I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/historia.rpy", line 17, in script
    jump pessimismo
ScriptError: could not find label 'pessimismo'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/historia.rpy", line 17, in script
    jump pessimismo
  File "C:\Users\Etec Paulino Botelho\Downloads\renpy-8.3.2-sdk\renpy\ast.py", line 1699, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\Etec Paulino Botelho\Downloads\renpy-8.3.2-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'pessimismo'.

Windows-10-10.0.19041 AMD64
Ren'Py 8.3.2.24090902
Trilhas do Brasil 1.0
Thu Nov  7 20:56:16 2024
